tests/mock_tests/Makefile.am

```diff
@@ -13,42 +15,48 @@ endif
-                ../orchagent/notifications.cpp \
-                ../orchagent/routeorch.cpp \
-                ../orchagent/neighorch.cpp \
-                ../orchagent/intfsorch.cpp \
-                ../orchagent/portsorch.cpp \
-                ../orchagent/copporch.cpp \
-                ../orchagent/tunneldecaporch.cpp \
-                ../orchagent/qosorch.cpp \
-                ../orchagent/bufferorch.cpp \
-                ../orchagent/mirrororch.cpp \
-                ../orchagent/fdborch.cpp \
-                ../orchagent/aclorch.cpp \
-                ../orchagent/saihelper.cpp \
-                ../orchagent/switchorch.cpp \
-                ../orchagent/pfcwdorch.cpp \
-                ../orchagent/pfcactionhandler.cpp \
-                ../orchagent/policerorch.cpp \
-                ../orchagent/crmorch.cpp \
-                ../orchagent/request_parser.cpp \
-                ../orchagent/vrforch.cpp \
-                ../orchagent/countercheckorch.cpp \
-                ../orchagent/vxlanorch.cpp \
-                ../orchagent/vnetorch.cpp \
-                ../orchagent/dtelorch.cpp \
-                ../orchagent/flexcounterorch.cpp \
-                ../orchagent/watermarkorch.cpp
+                $(top_srcdir)/orchagent/orchdaemon.cpp \
+                $(top_srcdir)/orchagent/orch.cpp \
+                $(top_srcdir)/orchagent/notifications.cpp \
+                $(top_srcdir)/orchagent/routeorch.cpp \
+                $(top_srcdir)/orchagent/neighorch.cpp \
+                $(top_srcdir)/orchagent/intfsorch.cpp \
+                $(top_srcdir)/orchagent/portsorch.cpp \
+                $(top_srcdir)/orchagent/copporch.cpp \
+                $(top_srcdir)/orchagent/tunneldecaporch.cpp \
+                $(top_srcdir)/orchagent/qosorch.cpp \
+                $(top_srcdir)/orchagent/bufferorch.cpp \
+                $(top_srcdir)/orchagent/mirrororch.cpp \
+                $(top_srcdir)/orchagent/fdborch.cpp \
+                $(top_srcdir)/orchagent/aclorch.cpp \
+                $(top_srcdir)/orchagent/saihelper.cpp \
+                $(top_srcdir)/orchagent/switchorch.cpp \
+                $(top_srcdir)/orchagent/pfcwdorch.cpp \
+                $(top_srcdir)/orchagent/pfcactionhandler.cpp \
+                $(top_srcdir)/orchagent/policerorch.cpp \
+                $(top_srcdir)/orchagent/crmorch.cpp \
+                $(top_srcdir)/orchagent/request_parser.cpp \
+                $(top_srcdir)/orchagent/vrforch.cpp \
+                $(top_srcdir)/orchagent/countercheckorch.cpp \
+                $(top_srcdir)/orchagent/vxlanorch.cpp \
+                $(top_srcdir)/orchagent/vnetorch.cpp \
+                $(top_srcdir)/orchagent/dtelorch.cpp \
+                $(top_srcdir)/orchagent/flexcounterorch.cpp \
+                $(top_srcdir)/orchagent/watermarkorch.cpp \
+                $(top_srcdir)/orchagent/chassisorch.cpp \
+                $(top_srcdir)/orchagent/sfloworch.cpp
```

Comment:
@stepanblyschak  @Pterosaur @nazariig @lguohan please make orchagent as library *.la, it will be one file, orchagent is already compiled, and here in tests you are compiling it again, which is twice the same compilation, it extend compilation time twice !, now it takes 19 minutes to compile and using it as a lib it could take 10 min